docs/backends/quickwit.mdx

```diff
@@ -0,0 +1,11 @@
+---
```

Comment:
I think this file should also be added to [mint.json](https://github.com/keyval-dev/odigos/blob/415c65d865343e902768bc9fdef95dc1ee2356bd/docs/mint.json#L65) for it to show up in the docs

Reply:
Done